letsencrypt-apache/letsencrypt_apache/configurator.py

```diff
@@ -493,7 +493,10 @@ def prepare_server_https(self, port, temp=False):
         """
         if "ssl_module" not in self.parser.modules:
             logger.info("Loading mod_ssl into Apache Server")
-            self.enable_mod("ssl", temp)
+            if self.config.func.__name__ == "auth":
```

Comment:
:) I agree... this solution was mostly chosen so I didn't have to deal with changing the CLI.

There is no current method to indicate to the plugin what the end goals are....
Install a certificate... vs get a certificate.

It would be great if there was a general method to get the verb off the command-line.

Will edit CLI and incur any additional fury it generates.